packages/components/src/theme/ThemeUtils.ts

```diff
@@ -294,45 +294,48 @@ export function replaceSVGFillColor(
 export function resolveCssVariablesInRecord<T extends Record<string, string>>(
   record: T,
   targetElement: HTMLElement = document.body,
-  isAlphaOptional = false
+  isAlphaOptional = true
 ): T {
   const perfStart = performance.now();
 
   // Add a temporary div to attach temp css variables to
   const tmpPropEl = document.createElement('div');
-  targetElement.appendChild(tmpPropEl);
-
-  const varExpressions = [...extractDistinctCssVariableExpressions(record)];
```

Comment:
Is this still able to resolve strings containing multiple expressions?
`var(--bbb1) var(--bbb2)`

Reply:
Yes.

```js
const el = document.createElement('div');
el.style.setProperty("--a", "#aaa");
el.style.setProperty("--b", "#bbb");
el.style.setProperty("--c", "var(--a) var(--b)");
document.body.appendChild(el);
window.getComputedStyle(el).getPropertyValue("--c");

// '#aaa #bbb'
```